profiles/kentik_snmp/citrix/netscaler.yml

```diff
@@ -144,6 +144,55 @@ metrics:
     symbol:
       name: totSpilloverCount
       OID: 1.3.6.1.4.1.5951.4.1.3.5.6.0
+  # Number of active SSL sessions
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: sslCurSessions
+      OID: 1.3.6.1.4.1.5951.4.1.1.47.296.0
+  # SSL sessions/second between client and NetScaler
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: sslSessionsPerSec
+      OID: 1.3.6.1.4.1.5951.4.1.1.47.3.0
+  # Current client connections in the Established state, which indicates that data transfer can occur between the NetScaler and the client.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpCurClientEstablished
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.12.0
+  # Port allocations that have failed on a mapped IP address because the maximum limit of 65536 has been exceeded.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpErrAnyPortFail
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.69.0
+  # Port allocations that have failed on a subnet IP address or vserver IP address because the maximum limit of 65536 has been exceeded.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpErrIpPortFail
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.70.0
+  # TCP packets retransmitted. The NetScaler attempts to retransmit the packet up to seven times, after which it resets the other half of the TCP connection.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpErrRetransmit
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.59.0
+  # TCP packets received.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpTotRxPkts
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.61.0
+  # TCP packets sent.
+  - MIB: NS-ROOT-MIB
+    symbol:
+      name: tcpTotTxPkts
+      OID: 1.3.6.1.4.1.5951.4.1.1.46.63.0
+  # This table contains information about the vServer statistics.
+  - MIB: NS-ROOT-MIB
+    table:
+      name: vserverTable
+      OID: 1.3.6.1.4.1.5951.4.1.3.1
+    symbols:
+      # Number of current client connections.
+      - name: vsvrCurClntConnections
+        OID: 1.3.6.1.4.1.5951.4.1.3.1.1.7
```

Comment:
this table needs a `metric_tag` section to support faceting in NR; recommend updating to the following:

```yaml
  # This table contains information about the vServer statistics.
  - MIB: NS-ROOT-MIB
    table:
      name: vserverTable
      OID: 1.3.6.1.4.1.5951.4.1.3.1
    symbols:
      # Current state of the server.
      - name: vsvrState
        OID: 1.3.6.1.4.1.5951.4.1.3.1.1.5
        enum:
          down: 1
          unknown: 2
          busy: 3
          outOfService: 4
          transitionToOutOfService: 5
          up: 7
          transitionToOutOfServiceDown: 8
      # Number of current client connections.
      - name: vsvrCurClntConnections
        OID: 1.3.6.1.4.1.5951.4.1.3.1.1.7
    metric_tags:
      # The name of the vserver
      - column:
          name: vsvrName
          OID: 1.3.6.1.4.1.5951.4.1.3.1.1.1
      # IP address of the vserver 
      - column:
          name: vsvrIpAddress
          OID: 1.3.6.1.4.1.5951.4.1.3.1.1.2
          conversion: hextoip
      # The port of the vserver
      - column:
          name: vsvrPort
          OID: 1.3.6.1.4.1.5951.4.1.3.1.1.3
      # Protocol associated with the vserver
      - column:
          name: vsvrType
          OID: 1.3.6.1.4.1.5951.4.1.3.1.1.4
          enum:
            http: 0
            ftp: 1
            tcp: 2
            udp: 3
            sslBridge: 4
            monitor: 5
            monitorUdp: 6
            nntp: 7
            httpserver: 8
            httpclient: 9
            rpcserver: 10
            rpcclient: 11
            nat: 12
            any: 13
            ssl: 14
            dns: 16
            adns: 17
            snmp: 18
            ha: 19
            monitorPing: 20
            sslOtherTcp: 21
            aaa: 22
            secureMonitor: 23
            sslvpnUdp: 24
            rip: 25
            dnsClient: 26
            rpcServer: 27
            rpcClient: 28
            dhcrpa: 36
            sipudp: 39
            siptcp: 40
            siptls: 41
            dnstcp: 48
            adnstcp: 49
            rtsp: 50
            push: 52
            sslPush: 53
            dhcpClient: 54
            radius: 55
            rdp: 61
            mysql: 62
            mssql: 63
            diameter: 73
            sslDiameter: 74
            tftp: 75
            tftp: 90
            pptp: 91
            gre: 92
            fix: 95
            sslFix: 96
            serviceUnknown: 99
```